### PR TITLE
Added modulo expression

### PR DESCRIPTION
**Pull request by Zekiah-A**
Slightly meh, but works
image

### CODE REVIEW

File: Nephrite/Lexer/Scanner.cs [hunk 1]
```diff
@@ -143,7 +143,16 @@ private void Scan()
                         AddToken(TokenType.Slash);
                         break;
                     }
-
+                case '%':
+                    if (Match('%'))
```
Comment (TheVeryStarlk) [inline Nephrite/Lexer/Scanner.cs]: What is this for?

**Comment by Zekiah-A** [comment]
Improved REPL message

**Comment by Zekiah-A** [comment]
image

File: Nephrite/Lexer/Scanner.cs (hunk 1 above)
Comment (Zekiah-A) [reply Nephrite/Lexer/Scanner.cs]: Lol can't remember, i don't think that it is needed actually :thinking:

Comment (TheVeryStarlk) [reply Nephrite/Lexer/Scanner.cs]: I edited it

**Pull request merged**